DivanteLtd/vue-storefront
# Next: Update nuxt theme module

## How to test
1. Prepare name for your create project directory. Let it be `new-project-test`
2. Open `package.json`
3. Add to scripts:
```
"test-cli": "cd packages/core/cli/src/scripts/createProject && ts-node index.ts"
```
4. Add to `workspaces.packages`:
```
"packages/new-project-test"
```
5. Open `packages/core/cli/src/scripts/createProject/index.ts`:
6. Add at the very bottom: 
```
createProject('commercetools', '../../../../../new-project-test')
```
7. Run `yarn test-cli` from root directory
8. `packages/new-project-test` should appear
9. Change name in package.json of new package to prevent conflict with existing in monorepo theme
10. Run `yarn` in root directory
11. `yarn dev` inside new package should work

## Things I tested
1. Unit tests
2. Core development with CT
3. Client development with created CT project

Closes #4652
